### DISCUSSION TITLE
How can I check the file Data after I selected it and validate for aspect ratio of image? Also How can I upload the image to AWS?

### DISCUSSION THREAD

**Discussion by Ruchika2224** [rpldy/react-uploady#378] [Q&A]
> Recommended - Check out the [discussions area](https://github.com/rpldy/react-uploady/discussions) before opening a new issue.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by yoavniran** [comment]
Hey @Ruchika2224 

sorry for the late reply, it has been a crazy period recently.

You can see this [sandbox](https://codesandbox.io/s/react-uploady-accept-filter-9ke9h?file=/src/App.js:212-224) for example of filtering based on file size. you can modify it to filter based on aspect-ratio.

You can also cancel uploads (if invalid aspect-ratio) with the [useItemStartListener](https://react-uploady.org/docs/api/hooks/useItemStartListener/)

you can see this discussion thread about uploading to S3 - https://github.com/rpldy/react-uploady/discussions/204

Im moving this to the [discussions](https://github.com/rpldy/react-uploady/discussions) area.
